import-dashboards.py

```diff
@@ -237,6 +237,22 @@ def add_panels():
                     print '   * Unzip %r' % (file,)
                     zip_ref.extractall(dest_dir)
                 os.remove(dest_dir + file)
+            rename_panels()
+
+
+def rename_panels():
+    for app in ['panels']:
+        print '  * Renaming %r' % (app,)
+        panels_dir = '/var/lib/grafana/plugins/'
+        panels_list = os.listdir(panels_dir)
+        for panel in panels_list:
+            print '   * %r -> ' % (panel),
```

Comment:
```suggestion
            print '   * %r -> ' % (panel,)
```
It is safer to use tuple here.

Reply:
btw, @adivinho, trailing comma is only required with there is one element:
```python
(one)       # single value with extra/bogus parentheses 
(one,)      # a tuple with a single value
(one, two)  # a tuple with two values
```

It is required there, but not in other PR where it leads to confusion.